core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/footer.jelly

```diff
@@ -0,0 +1,159 @@
+            position: absolute;
+            bottom: -5px;
+            left: 32%;
+            width: 0;
+            height: 0;
+            border-left: 7px solid transparent;
+            border-right: 7px solid transparent;
+            border-bottom: 7px solid #fff;
+            opacity: 0;
+            transition: all .05s;
+        }
+        #visible-am-container.visible #visible-am-button:after {
+            opacity: 1;
+            bottom: 4px;
+            transition: all .25s;
+        }
+        #visible-am-container .am-message {
+            display: block;
+            line-height: 1.4em;
+            margin-bottom: 1.4em;
+        }
+        </style>
+        <script>
+            function amCloser(e) {
+                var list = document.getElementById('visible-am-list');
+                var el = e.target;
+                while (el) {
+                    if (el === list) {
+                        return; // clicked in the list
+                    }
+                    el = el.parentElement;
+                }
+                hideVisibleAmList();
+            };
+            function amEscCloser(e) {
+                if (e.keyCode == 27) {
+                    amCloser(e);
+                }
+            };
+            function amContainer() {
+                return document.getElementById('visible-am-container');
+            };
+            function hideVisibleAmList(e) {
+                amContainer().classList.remove('visible');
+                document.removeEventListener('click', amCloser);
+                document.removeEventListener('keydown', amEscCloser);
+            }
+            function showVisibleAmList(e) {
+                amContainer().classList.add('visible');
+                setTimeout(function() {
+                    document.addEventListener('click', amCloser);
+                    document.addEventListener('keydown', amEscCloser);
+                }, 1);
+            }
+            function toggleVisibleAmList(e) {
+                if (amContainer().classList.contains('visible')) {
+                    hideVisibleAmList(e);
+                } else {
+                    showVisibleAmList(e);
+                }
+                e.preventDefault();
+            }
+        </script>
+        <div id="visible-am-container">
+            <a id="visible-am-button" href="#" onclick="toggleVisibleAmList(event)">
+                ${it.activeAdministrativeMonitorsCount}
+            </a>
+            <div id="visible-am-list">
+                <j:forEach var="am" items="${it.activeAdministrativeMonitors}">
+                    <div class="am-message">
+                    <st:include page="message.jelly" it="${am}" />
+                    </div>
+                </j:forEach>
+                <p style="text-align: center; margin: 10px 0 0 0;">
+                    <a href="${rootURL}/manage" onclick="document.location.href='${rootURL}/manage';">${%Manage Jenkins}</a>
```

Comment:
Can anyone guess what the purpose of this may have been? I have no idea what I thought when I did that.